src/e2e/java/teammates/e2e/util/BackDoor.java

```diff
@@ -258,6 +259,22 @@ public static CourseAttributes getCourse(String courseId) {
         return CourseAttributes.builder(courseData.getCourseId()).build();
     }
 
+    /**
+     * Gets the ID of an feedback question from the datastore.
+     */
+    public static String getFeedbackQuestionId(String courseId, String fsName, int questionNumber) {
+        Map<String, String[]> params = new HashMap<>();
+        params.put(Const.ParamsNames.COURSE_ID, new String[] { courseId });
+        params.put(Const.ParamsNames.FEEDBACK_SESSION_NAME, new String[] {fsName});
+        params.put(Const.ParamsNames.INTENT, new String[] {"INSTRUCTOR_RESULT"});
+        ResponseBodyAndCode response = executeGetRequest(Const.ResourceURIs.QUESTIONS, params);
```

Comment:
As you make implicit assumption that an instructor is logged in.

Reply:
I chose the "INSTRUCTOR_RESULT" intent because it provides the feedback questions without checking for logged in instructors, as shown in the code here (GetFeedbackQuestionsAction.java, line 78).
```
            case INSTRUCTOR_RESULT:
                questions = logic.getFeedbackQuestionsForSession(feedbackSessionName, courseId);
```
Plus the Backdoor api access gives us full privilege, which passes the privilege check. It minimized the amount of change I needed to make to make this temporary backdoor workaround work.

Reply:
Added documentation to explain the situation in `getFeedbackQuestionId` method